### ISSUE TITLE
Update map pop-ups on desktop experience

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#895]
<img width="1261" alt="screen shot 2017-08-30 at 10 17 24 am" src="https://user-images.githubusercontent.com/5550969/29880085-86ee8d74-8d6c-11e7-8b59-66ff1a7c3a1d.png">

- [x] We want the pop-ups to contain similar information that exists on our live site: name of site, address, Full/Part Day, Full/School Year, CPS-based/Community-based, If Early Headstart/Headstart
- [ ] Add a "+ Compare" link or button

**Comment by smarziano** [comment]
I am separating the "+" Compare icon to these pop-ups into a GitHub separate issue. The first part of this issue is waiting on merge.
